lib/sprangular.rb

```diff
@@ -11,6 +11,7 @@
 require 'rails-assets-angular-route'
 require 'rails-assets-angular-sanitize'
 require 'rails-assets-angular-animate'
+require 'rails-assets-ngInfiniteScroll'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.